### ISSUE TITLE
Upload image/zip files to dataset

### ISSUE THREAD

**Issue by jsbroks** [jsbroks/coco-annotator#29]

**Comment by jsbroks** [comment]
No it is not

**Comment by jsbroks** [comment]
I am looking for contributors, if you would like to help out.

**Comment by gachiemchiep** [comment]
@jsbroks 
Ok i will take a look at this.

**Comment by jsbroks** [comment]
@gachiemchiep Any update on progress?

**Comment by gachiemchiep** [comment]
@jsbroks 
Sorry man. My computer got broken so i can't work from home yet.
I am going to start from next week.

**Comment by NicksonYap** [comment]
@jsbroks 
I suppose uploads will need to be implement on Flask on Server-side and Vue + Axios on Frontend?

**Comment by jsbroks** [comment]
Yes. Could probably use an existing vue component

**Comment by winfish** [comment]
Hello jsbroks,
I was able to modified to see like this:
![2019-07-31 14_47_11-COCO Annotator](https://user-images.githubusercontent.com/28815305/62243095-36625a80-b3a2-11e9-86eb-57fce65faf7e.png)

And:
![2019-07-31 14_47_21-COCO Annotator](https://user-images.githubusercontent.com/28815305/62243152-4f6b0b80-b3a2-11e9-869e-5cdec1acd7dd.png)

Is this seems what we need ?

**Comment by jsbroks** [comment]
Yes, could you add the button inside the dataset view? Not from the dropdown window

**Comment by winfish** [comment]
I could try. Current the most problem I have is how to debug regards for this project. Everytime I change the code I have to remove all four containers and rerun the command to docker-compose build to see my coding changes. Do you have a better way that can share me that I can more convenient programming and help contributing this project?

**Comment by jsbroks** [comment]
@winfish are you using the dev docker build? Are you developing on Windows or Linux?

**Comment by winfish** [comment]
I am developing in Linux and use the command of "docker-compose -f docker-compose.build.yml up --build"

**Comment by jsbroks** [comment]
Yes, it should update as you change the files, the web client and backend will be on two different ports

**Comment by jsbroks** [comment]
You won't have to recompile the container, at the worse restart it

**Comment by winfish** [comment]
Thanks, It works and helps. 
I have made a frondend change. Need more programming to create the javascript and post to the backend.
![image](https://user-images.githubusercontent.com/28815305/62386424-6d10b000-b51d-11e9-8439-94ecb9d4b763.png)

**Comment by jsbroks** [comment]
Looks good!

**Comment by skywalkerisnull** [comment]
@winfish, have you made any more progress? Or is there something that I might be able to assist with?

**Comment by winfish** [comment]
I will work on it recently. Maybe need help from you
Thanks.

________________________________
